core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -639,6 +639,8 @@ public enum BuiltInMethod {
   ARRAY_INTERSECT(SqlFunctions.class, "arrayIntersect", List.class, List.class),
   ARRAY_UNION(SqlFunctions.class, "arrayUnion", List.class, List.class),
   ARRAY_REVERSE(SqlFunctions.class, "reverse", List.class),
+  ARRAYS_OVERLAP(SqlFunctions.class, "arraysOverlap", List.class, List.class),
```

Comment:
Why no entry for `ARRAY_JOIN`?

Reply:
@tanclary       defineMethod(ARRAY_JOIN, "arrayToString", NullPolicy.STRICT), which  aligns with array_to_string in bigquery